source/help/getting-started/organizing-conversations.rst

```diff
@@ -46,40 +46,54 @@ Channels can be created, joined, renamed, left, marked as a favorite, muted, unm
 Creating a channel
 ~~~~~~~~~~~~~~~~~~~
 
-Create a new Public channel or Private channel by selecting the **+** symbol next to the **Public Channels*** or **Private Channels** header on the left hand navigation. To start a Direct Message thread, select the **+** symbol next to the **Direct Messages** bar or select **More** at the bottom of the list to view team members you can message.
-
-    .. image:: ../../images/Create_private_channel.png
+Create a new Public or Private channel by selecting the **+** symbol at the top of the sidebar, next to the **History** arrows, then selecting **Create New Channel**.
 
+    .. image:: ../../images/create-new-channel.png
+    
 Anyone can create Public channels or Private channels, unless the System Admin has `restricted the permissions <https://docs.mattermost.com/administration/config-settings.html#enable-public-channel-creation-for>`__.
 
+There are two ways to start a Direct or Group Message:
+
+Select the **+** symbol at the top of the sidebar, next to the **History** arrows, then select **Open a Direct Message**.
+
+or
+
+Select the **+** symbol next to the **Direct Messages** category in the sidebar.
+
+    .. image:: ../../images/write-dm.png
+
 Joining a channel
 ~~~~~~~~~~~~~~~~~~
 
-Select **More** at the bottom of the **Public Channels** list to view and search through a list of Public channels you can join. To join a Private channel you need to be added by a member of that channel.
+Join a Public channel by selecting the **+** symbol at the top of the sidebar, next to the **History** arrows, then selecting **Browse Channels**. Search through the list of Public channels you can join, then select **Join**.
+
+To join a Private channel, you need to be added to the channel by a member of that channel.
 
 Adding members to a channel
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Select the channel name at the top of the center pane to access the drop-down menu, then select **Add Members**. Any member of a channel can add new members by choosing **Add** next to a user's name. Users already added to the channel will not appear in this list.
 
-    .. image:: ../../images/add_members.png
+    .. image:: ../../images/add-member-to-channel.png
```

Comment:
image need updating. Let's avoid using the red squares and just use the in-product hover effects to highlight the element of interest

Reply:
Agreed on the usage in-product hover effects over red square labelling. This image was has been updated in a previous commit.